docs/src/api/class-consolemessage.md

```diff
@@ -5,6 +5,25 @@
 ## method: ConsoleMessage.args
 - returns: <[Array]<[JSHandle]>>
 
+List or arguments passed to a `console` function call.
+
+```js
+const { chromium } = require('playwright');  // Or 'firefox' or 'webkit'.
+
+(async () => {
+  const browser = await chromium.launch();
+  const page = await browser.newPage();
+  await Promise.all([
+    page.evaluate(() => console.log('hello', 5, {foo: 'bar'})),
+    page.waitForEvent('console')
+  ]);
+  console.log(await message.args()[0].jsonValue()); // It will print 'hello'
+  console.log(await message.args()[1].jsonValue()); // It will print 5
+  console.log((await message.args()[2].jsonValue()).foo); // It will print 'bar'
+  await browser.close();
+})();
```

Comment:
```suggestion
const { chromium } = require('playwright');  // Or 'firefox' or 'webkit'.

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  const [message] = await Promise.all([
    page.waitForEvent('console'),
    page.evaluate(() => console.log('hello', 5, { foo: 'bar' })),
  ]);
  console.log(await message.args()[0].jsonValue()); // It will print 'hello'
  console.log(await message.args()[1].jsonValue()); // It will print 5
  console.log((await message.args()[2].jsonValue()).foo); // It will print 'bar'
  await browser.close();
})();
```

See here: https://try.playwright.tech/?l=javascript&s=5qv6x1x

Reply:
applied and rebuilt to make the ci happy